config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,58 @@
-create table bad_words (
-    word character varying(40) not null
+CREATE TABLE ta_users (
+    username character varying(40) NOT NULL PRIMARY KEY,
+    password character(34),
+    bcrypt_password character(60),
+    CHECK (password IS NOT NULL OR bcrypt_password IS NOT NULL),
+    email character varying(254) NOT NULL,
+    joined timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
+    lastlogin timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
+    admin integer NOT NULL DEFAULT 0
 );
+ALTER TABLE ta_users owner TO postgres;
 
-
-alter table bad_words owner to postgres;
-
-
-create table banned_ips (
-    ip character varying(40) not null,
-    ban_till timestamp without time zone
+CREATE TABLE bad_words (
+    word character varying(40) NOT NULL PRIMARY KEY
```

Comment:
I think a Database is overkill for a single collection like this.
In theory this table shouldn't be even changed that often, so IMO a space separated file would be sufficient, we could simply update it here on GitHub

Reply:
The benefit of live updates is pretty significant. A file you would need to re-read each time. The file requires direct server access and a file to be modified, previously it was all set up so moderators could use the a GUI to update DB values. 

I personally would like us to stick to exactly one configuration file for now if we can get away with it.

On the other hand, we are missing the data for this table and need some insert statements.